public/package.json

```diff
@@ -3,9 +3,11 @@
   "version": "0.2.0",
   "private": true,
   "dependencies": {
-    "@bufbuild/connect": "^0.8.4",
-    "@bufbuild/connect-web": "^0.8.4",
-    "@bufbuild/protobuf": "^1.2.0",
+    "@bufbuild/protobuf": "^2.2.0",
+    "@bufbuild/protoc-gen-es": "^2.2.0",
+    "@connectrpc/connect": "^2.0.0-rc.1",
```

Comment:
Looks like there is now a released version of these;
- https://www.npmjs.com/package/@connectrpc/connect
- https://www.npmjs.com/package/@connectrpc/connect-web